docs/content/controls/_gallery.md

```diff
@@ -0,0 +1,49 @@
+#Gallery
+A control that allows the user to choose set of images from the WordPress media library. 
+
+*Returns:* An array of single image arrays:
+```php
+array(
```

Comment:
The whole plugin uses the square brance format for arrays; I think docs should reflet the same code style.